os/fs/driver/mtd/smart.c

```diff
@@ -521,6 +525,28 @@ static const struct block_operations g_bops = {
  ****************************************************************************/
 
 #ifdef CONFIG_DEBUG_FS
+#ifndef NXFUSE_HOST_BUILD
+static int smart_dumpext(FAR struct smart_struct_s *dev)
+{
+	int ret = OK;
```

Comment:
I think we don't need to initialize the `ret`. Because it is always set before using.

Reply:
I follow this because I think it is a good practice to safeguard from garbage initialisation.
I'd like to keep this line same.